### PR TITLE
openshift-psap/ci-artifacts: Rename tests for ansible lightspeed

### PR DESCRIPTION
**Pull request by dagrayvid**

### CODE REVIEW

File: ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml [hunk 1]
```diff
@@ -264,6 +264,172 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (kpouget) [inline ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml:268]: can you disable all the `always_run: true`?

File: ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main__lightspeed.yaml [hunk 2]
```diff
@@ -40,7 +40,7 @@ tests:
         export WISDOM_SECRET_PATH=/var/run/psap-wisdom
         export WISDOM_PROTOS_SECRET_PATH=/var/run/psap-wisdom-protos
 
-        run ansible-llm load_test prepare
+        run lightspeed test prepare
```
Comment (kpouget) [inline ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main__lightspeed.yaml:43]: can you `prepare_ci` and `test_ci` entrypoint names?

the idea behind the `_ci` suffix is that `prepare` and `test` should be runnable from the laptop, 
while the `*_ci` entrypoint are reserved for any CI-specific things that should be done before

Comment (dagrayvid) [reply ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main__lightspeed.yaml]: Makes sense, done.

File: ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml (hunk 1 above)
Comment (dagrayvid) [reply ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml]: Thanks, done.

**Comment by kpouget** [comment]
thanks @dagrayvid 
/lgtm
/approve

**Pull request merged**